Extensions/PlatformBehavior/Extension.cpp

```diff
@@ -583,6 +583,19 @@ void DeclarePlatformBehaviorExtension(gd::PlatformExtension& extension) {
         .MarkAsAdvanced()
         .SetFunctionName("GetCurrentJumpSpeed");
 
+    aut.AddScopedAction("SetCurrentSpeed",
+                     _("Current horizontal speed"),
+                     _("Change the current horizontal speed of the object (in pixels per "
```

Comment:
This horizontal speed is different from other speeds as it has the notion of direction (i.e: it's not absolute). I think we should somehow call this out in the naming/description?

Reply:
Yes, we could add "The object moves to the left with negative values and to the right with positive ones" in the description of action, condition and expression.

I'll do a PR to change this and remove the parameter from the PR.